test/junit/scala/tools/nsc/backend/jvm/opt/UnusedLocalVariablesTest.scala

```diff
@@ -35,7 +35,7 @@ class UnusedLocalVariablesTest extends BytecodeTesting {
     assertLocalVarCount(code, 6)
 
     val code2 = """def f(a: Long): Unit = { var x = if (a == 0l) return else () }"""
-    assertLocalVarCount(code2, 2)
```

Comment:
Not my finest hour: I incorrectly changed the assertion in this test when it failed in #8897

Reply:
... and I wrote a smart sounding explanation to back up the change